src/main/cc/wfa/virtual_people/core/model/utils/distributed_consistent_hashing.cc

```diff
@@ -0,0 +1,83 @@
+// Copyright 2021 The Cross-Media Measurement Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "wfa/virtual_people/core/model/utils/distributed_consistent_hashing.h"
+
+#include <math.h>
```

Comment:
Please include cmath and cstdlib and prefix mathematical functions with "std::". There are some really subtle bugs that can come from math.h.

Reply:
Done.